### PR TITLE
feat: Initial version.

### PR DESCRIPTION
**Pull request by ShogunPanda**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### 2020-03-31 / 0.1.0
+
+- Initial version.
```
Comment (mcollina) [inline CHANGELOG.md]: no need for this file.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@nearform/heap-profiler",
+  "version": "0.1.0",
```
Comment (mcollina) [inline package.json]:
```suggestion
  "version": "0.0.1",
```

Bump a minor directly on master and release.

File: src/snapshot.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+  // Open the destination file
+  try {
+    fd = openSync(destination, 'w')
+  } catch (e) {
+    return callback(e)
+  }
+
+  // Start the session
+  session.connect()
```
Comment (mcollina) [inline src/snapshot.js]: can session emit 'error'? In case, we should call the calback

File: test/index.spec.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+const t = require('tap')
```
Comment (mcollina) [inline test/index.spec.js]: use strict is missing

File: package.json (hunk 2 above)
Comment (ShogunPanda) [reply package.json]: Will do.

File: src/snapshot.js (hunk 3 above)
Comment (ShogunPanda) [reply src/snapshot.js]: Apparently no, it cannot.

File: test/index.spec.js (hunk 4 above)
Comment (ShogunPanda) [reply test/index.spec.js]: Added.

File: CHANGELOG.md (hunk 1 above)
Comment (ShogunPanda) [reply CHANGELOG.md]: Removed.

**Pull request merged**